remote-config.jsonc

```
@@ -153,7 +153,7 @@
           "message": "`ddev config --update` will change your project config where possible based on the code that is discovered in your project."
         },
         {
-          "message": "Please remove `router_http_port`, `router_https_port`, `project_tld`, and `nodejs_version` from a project’s `config.yaml` so it can use the global defaults."
+          "message": "Did you know `router_http_port`, `router_https_port`, `project_tld`, and `nodejs_version` are set in global defaults, you may not need them in your project anymore."
```

```suggestion
          "message": "Did you know `router_http_port`, `router_https_port`, `project_tld`, and `nodejs_version` are set in global configuration, and most projects will benefit from removing them from the project configuration and handling them in global config?"
```

---

```suggestion
          "message": "Did you know `router_http_port`, `router_https_port`, `project_tld`, and `nodejs_version` are now set in global configuration? Consider removing them from your project configuration.",
```

Having project level configuration is allowed right,

Since its pretty common to not have every single project instantly move to a new version of something... they upgrade over time, where I'm going with this is that it takes time to adopt the "standard" (assuming you'd have your global config to the "standard".
Thus you wouldn't want the language to imply you _**have**_ to or **_must_** change away.

Also how is it a **benefit**? That's a pretty loaded term. 🤔 

I can see removing `project_tld` as nice as there will be less config now...
but say there are some node projects still on 18, some on 20.0.0 some on others...


What about?
```
"Did you know `router_http_port`, `router_https_port`, `project_tld`, and `nodejs_version` are now set in global configuration? Consider removing them from your project configuration."
```

---

Yes, project-level config of all of these is fine. It's very unusual for the ports or the project_tld to differ per project. 

The motivation behind this is that these were once *only* at the project level, but people complained about that because they wanted to have them global. So we added global config with project overrides. But now on projects that have been around for awhile people don't know that, and the global config doesn't work for them because they have the project overrides built in from when their project was originally configured.

I guess it's possible we should just remove this whoe line now since time has moved on and newer projects don't have that old automatic `router_http_port` in there. Or the explicit nodejs_version.

---

🤔 
Yeah.

Language needs to consider the following perspectives.

1. new to DDEV
2. experienced with DDEV
  a. old project
  b. new project
  
 
Makes more sense to have a tip that implies `DDEV have helpful global default settings. You can override them on a project level if you'd like` yeah?

---

How about:

> The `project_tld, nodejs_version`, `router_http_port`, and `router_https_port` configuration can all be set globally. If you have multiple projects which are setting these values independently in their config.yaml files, consider setting them globally instead.

This puts the two more common configs up front, and makes it clear that they _can_ be set globally, and recommends considering doing that.
If you want a more strongly worded recommendation, then something like this might work:


> The `project_tld, nodejs_version`, `router_http_port`, and `router_https_port` configuration can all be set globally. This avoids duplication and means you only have to change the configuration in one place rather than for every project. If you have multiple projects which are setting these values independently in their config.yaml files, we recommend setting them globally instead.

---

If you're not so concerned about the router port configs anymore those could be ommitted - or, if you think it's been long enough that most people will have already made the change, removing this entry altogether doesn't seem like a bad option.

---

I quite like
> The project_tld, nodejs_version, router_http_port, and router_https_port configuration can all be set globally. This avoids duplication and means you only have to change the configuration in one place rather than for every project. If you have multiple projects which are setting these values independently in their config.yaml files, we recommend setting them globally instead.

---

With your approval, can we just remove that line? If it causes this much confusion, we just don't need it. Thanks so much for piping up about the need there. I don't think we're going to be able to communicate what we need to communicate in just a few words. I pushed commit to remove.